tensorflow/tools/ci_build/osx/arm64/tensorflow_as_test_ci.Jenkinsfile

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+pipeline {
+    agent none
+    stages {
+        stage("Build Tensorflow") {
+            parallel {
+                stage("Python 3.8") {
+                    agent {
+                        label "silicon-ci"
+                    }
+                    environment {
+                        PYENV_ROOT="$HOME/.pyenv"
+                        PATH="$PYENV_ROOT/shims:/opt/homebrew/bin/:$PATH"
+
+                    }
+                    steps {
+
+                        sh '''
+                            echo 3.8.13 > /Users/admin/.python-version
+                            pyenv init -
+                            pyenv global 3.8.13
+                        '''
+
+                        sh 'python --version'
+
+                        git branch: "nightly",
```

Comment:
Sorry if https://github.com/tensorflow/tensorflow/pull/60575#discussion_r1192630941 was confusing. 

What I meant was we need three versions of this file for three separate builds:
1. Continuous builds - Runs from the master branch at an interval of every 3 hours. Ideally, it would be nice to have it run for each new commit to the master branch but the macOS builds are not fast enough for it to be a real option.
2. Nightly builds - Runs once per night from the nightly branch sometime between 12AM to 3AM.  
3. Release builds - Runs from head of the release branch and is triggered on an ad-hoc basis.